packages/domains-table/src/domains-table/domains-table.tsx

```diff
@@ -147,8 +150,9 @@ export const DomainsTable = ( props: DomainsTableProps ) => {
 		children,
 		onDomainAction,
 		userCanSetPrimaryDomains,
-		shouldDisplayContactInfoBulkAction = false,
+		shouldDisplayContactInfoBulkAction = true,
```

Comment:
Let's revert this before merging? I think it was introduced for testing, but you can enable it anyway through the feature flag.